src/builtins.jl

```diff
@@ -228,7 +229,7 @@ function maybe_evaluate_builtin(frame, call_expr, expand::Bool)
         cargs = getargs(args, frame)
         return Some{Any}(ccall(:jl_f_intrinsic_call, Any, (Any, Ptr{Any}, UInt32), f, cargs, length(cargs)))
     end
-    if isa(f, getfield(Core, Symbol("#kw##invoke")))
+    if isa(f, getfield(Core, kwinvoke_name))
```

Comment:
`getfield(Core, kwinvoke_name)` -> `kwinvoke_instance`?

Reply:
Hm, I don't think so.

Reply:
Oh, true, sorry. Doesn't matter much either way, it's not like a `getfield` is going to make a performance difference or anything.